src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -448,59 +455,62 @@ const initialise = (window: Window, consentState: ConsentState): void => {
 	if (shouldInclude('criteo')) {
 		window.pbjs.bidderSettings.criteo = {
 			storageAllowed: true,
+			// Use a custom price granularity, which is based upon the size of the slot being auctioned
+			adserverTargeting: [
+				{
+					key: 'hb_pb',
+					val({ width, height, cpm, pbCg }) {
+						return overridePriceBucket(
+							'criteo',
+							width,
+							height,
+							cpm,
+							pbCg,
+						);
+					},
+				},
+			],
 		};
-
-		// Use a custom price granularity for Criteo
-		// Criteo has a different line item structure and so bids should be rounded to match these
-		window.pbjs.setBidderConfig({
-			bidders: ['criteo'],
-			config: {
-				customPriceBucket: criteoPriceGranularity,
-			},
-		});
 	}
 
 	if (shouldInclude('ozone')) {
 		// Use a custom price granularity, which is based upon the size of the slot being auctioned
-		window.pbjs.setBidderConfig({
-			bidders: ['ozone'],
-			config: {
-				// Select the ozone granularity, use default if not defined for the size
-				guCustomPriceBucket: ({ width, height }) => {
-					const ozoneGranularity = ozonePriceGranularity(
-						width,
-						height,
-					);
-					log(
-						'commercial',
-						`Custom Prebid - Ozone price bucket for size (${width},${height}):`,
-						ozoneGranularity,
-					);
-					return ozoneGranularity;
+		window.pbjs.bidderSettings.ozone = {
+			adserverTargeting: [
+				{
+					key: 'hb_pb',
+					val({ width, height, cpm, pbCg }) {
```

Comment:
While this works but I find this syntax a little 'exotic'.

Might this be a little less surprising?

```suggestion
					val: ({ width, height, cpm, pbCg }) => {
```